code/modules/mob/living/simple_animal/hostile/mining/goliath.dm

```diff
@@ -188,3 +188,20 @@
 	icon_state = "Goliath_tentacle_retract"
 	deltimer(timerid)
 	timerid = QDEL_IN(src, 5)
+
+/mob/living/simple_animal/hostile/asteroid/goliath/beast
+	name = "goliath"
+	desc = "A hulking, armor-plated beast with long tendrils arching from its back."
+	icon = 'icons/mob/lavaland/lavaland_monsters.dmi'
+	icon_state = "goliath"
+	icon_living = "goliath"
+	icon_aggro = "goliath"
+	icon_dead = "goliath_dead"
+	throw_message = "does nothing to the tough hide of the"
+	pre_attack_icon = "goliath2"
+	butcher_results = list(/obj/item/reagent_containers/food/snacks/goliath = 2, /obj/item/stack/sheet/animalhide/goliath_hide = 1, /obj/item/stack/sheet/bone = 2)
+	loot = list()
+	crusher_drop_mod = 25
```

Comment:
I feel like the ancient goliath should either have a guaranteed chance to drop the tentacle, or perhaps some special loot of its own?